+ source ../common/storage_disk_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Wed Sep 27 14:20:24 2023 - INFO  - Start environment preparation.
+ check_free_disk 2
++ TEST_DISK
++ id=1
++ python3 /root/mugen/libs/locallibs/get_test_device.py --device disk --node 1
+ disks='fd0 sr0 vdb'
+ disk_list=($disks)
+ local_disk=fd0
+ local_disk1=sr0
+ local_disk2=vdb
+ local_disk3=
+ mkfs.ext4 -F /dev/fd0
mke2fs 1.47.0 (5-Feb-2023)
mkfs.ext4: No such device or address while trying to determine filesystem size
+ mkfs.ext4 -F /dev/sr0
mke2fs 1.47.0 (5-Feb-2023)
mkfs.ext4: No medium found while trying to determine filesystem size
+ pvcreate -y /dev/fd0 /dev/sr0
oe_test_storage_lvm_set_label.sh: line 25: pvcreate: command not found
+ vgcreate openeulertest /dev/fd0 /dev/sr0
oe_test_storage_lvm_set_label.sh: line 26: vgcreate: command not found
+ lvcreate -L 200M -n test openeulertest --persistent y --major 253 --minor 7788 -y
oe_test_storage_lvm_set_label.sh: line 27: lvcreate: command not found
++ pvs -o+pv_used
++ wc -l
+ number=1
+ LOG_INFO 'Environmental preparation is over.'
+ message='Environmental preparation is over.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Environmental preparation is over.'
Wed Sep 27 14:20:24 2023 - INFO  - Environmental preparation is over.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase!'
+ message='Start executing testcase!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase!'
Wed Sep 27 14:20:24 2023 - INFO  - Start executing testcase!
+ test 1 -gt 3
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_set_label.sh line 35'
+ message='oe_test_storage_lvm_set_label.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_set_label.sh line 35'
Wed Sep 27 14:20:24 2023 - ERROR - oe_test_storage_lvm_set_label.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ lvchange --persistent y --major 253 --minor 7788 openeulertest/test -y
oe_test_storage_lvm_set_label.sh: line 36: lvchange: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_set_label.sh line 37'
+ message='oe_test_storage_lvm_set_label.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_set_label.sh line 37'
Wed Sep 27 14:20:24 2023 - ERROR - oe_test_storage_lvm_set_label.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End of testcase execution!'
+ message='End of testcase execution!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of testcase execution!'
Wed Sep 27 14:20:24 2023 - INFO  - End of testcase execution!
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Sep 27 14:20:25 2023 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Wed Sep 27 14:20:25 2023 - INFO  - start environment cleanup.
+ lvremove openeulertest/test -y
oe_test_storage_lvm_set_label.sh: line 43: lvremove: command not found
+ vgremove openeulertest -y
oe_test_storage_lvm_set_label.sh: line 44: vgremove: command not found
+ pvremove /dev/fd0 /dev/sr0 -y
oe_test_storage_lvm_set_label.sh: line 45: pvremove: command not found
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Wed Sep 27 14:20:25 2023 - INFO  - Finish environment cleanup.
